chain/client/src/stateless_validation/chunk_validator/mod.rs

```diff
@@ -290,8 +291,14 @@ pub(crate) fn pre_validate_chunk_state_witness(
     }
 
     let main_transition_params = if last_chunk_block.header().is_genesis() {
+        // TODO: check if this epoch_id / protocol version is right for genesis
+        let epoch_id = epoch_manager
+            .get_next_epoch_id_from_prev_block(last_chunk_block.header().prev_hash())?;
```

Comment:
I think the block or block header has epoch id in it, it should be safe to use that one.